docs/docker/Dockerfile.production

```diff
@@ -22,7 +22,7 @@ WORKDIR /app/docs
 
 RUN rm -rf ./next/
 # Removes the symlink to the "next" Handsontable version.
-RUN rm ./.vuepress/public/handsontable
+# RUN rm ./.vuepress/public/handsontable # todo causes an error `rm: cannot remove './.vuepress/public/handsontable': No such file or directory` (@see https://github.com/handsontable/handsontable/runs/3180829710?check_suite_focus=true#step:4:157)
```

Comment:
I guess it's not going to be a problem now (because the symlink doesn't exist, as the error states), but we should probably tag @budnix here, for future reference.

Reply:
I guess, that the symlink is created during the build process latter (by a vuepress plugin wroten recently). Anyway, the effect of existing this symlink is only serving handsontable's `./dist` folder.